src/pages/workspaces/workspace/_tests/WorkspaceMenu.test.js

```diff
@@ -0,0 +1,258 @@
+import '@testing-library/jest-dom'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { MenuTrigger } from 'src/components/PopupTrigger'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import { useWorkspaceDetails } from 'src/components/workspace-utils'
+import WorkspaceMenu from 'src/pages/workspaces/workspace/WorkspaceMenu'
+
+
+jest.mock('src/components/workspace-utils', () => {
+  const originalModule = jest.requireActual('src/components/workspace-utils')
+  return {
+    ...originalModule,
+    useWorkspaceDetails: jest.fn()
+  }
+})
+
+jest.mock('src/components/PopupTrigger', () => {
+  const originalModule = jest.requireActual('src/components/PopupTrigger')
```

Comment:
lets put a comment on this mock and the TooltipTrigger mock  below to state that we are mocking them to avoid test environment issues with React Portal's requirement to use DOM measure services which are not available in jest env.